bifacial_radiance/main.py

```diff
@@ -978,7 +978,7 @@ def readWeatherFile(self, weatherFile=None, starttime=None,
                             coerce_year=coerce_year, label=label,
                             tz_convert_val=tz_convert_val)
         
-        return self.metdata
+        return self.metdata, metadata
```

Comment:
@mjprilliman  - you need to revert this line - other functions are expecting a br.MetObj return here, not a tuple.  If you need other raw values from metadata, let's include it in the MetObj, but most of the relevant metadata details should already be included directly accessible in the MetObj.   Maybe we can include an additional MetObj attribute like `metadata_raw` that stores the original raw metadata?